docs-gh-pages/consensus.md

```diff
@@ -0,0 +1,91 @@
+## Consensus
+
+### Architecture
+Every new `CheckpointBlock` is being created under a consensus round.
+
+### Glossary
+- **consensus owner** - the node which sent consensus proposal to others and initiated a new round
+- **facilitator** - node which has been selected to take part in a new round and responded successfully to participate in it (including owner)
+- **SOE** - signed observation edge
+- **data proposal** - transactions (and other checkpoint block data) proposed by the facilitator to include in the new consensus round
+
+### Consensus round
+1. Start consensus
+2. Broadcast and gather consensus data proposals
+3. Merge, broadcast and gather consensus block proposals
+4. Select majority block proposal and merge facilitators' signatures
```

Comment:
Majority or Union?

Reply:
it selects majority block proposal and create a union block (merged signatures)